### PR TITLE
Add forum URL and update first steps using ES and Gradle

### PR DESCRIPTION
**Pull request by ayusopatricia**
Add forum URL so new contributors can refer to it, and update first steps using ES and Gradle in CONTRIBUTING documentation.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -134,17 +134,20 @@ onwards and Eclipse 2020-3 and onwards.
     ./gradlew :run
 
 That'll spend a while building Elasticsearch and then it'll start Elasticsearch,
-writing its log above Gradle's status message. We log a lot of stuff on startup,
-specifically these lines tell you that Elasticsearch is ready:
+writing its log above Gradle's status message. We log a lot of stuff on startup, when you see some lines like these ones (when the console stops scrolling):
```
Comment (henningandersen) [inline CONTRIBUTING.md:137]: Can we limit this to 80 characters line width?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -134,17 +134,20 @@ onwards and Eclipse 2020-3 and onwards.
...
 
-    [2020-05-29T14:50:35,167][INFO ][o.e.h.AbstractHttpServerTransport] [runTask-0] publish_address {127.0.0.1:9200}, bound_addresses {[::1]:9200}, {127.0.0.1:9200}
-    [2020-05-29T14:50:35,169][INFO ][o.e.n.Node               ] [runTask-0] started
```
Comment (henningandersen) [inline CONTRIBUTING.md:141]: I see that we end with the security line now. It is a bit arbitrary and somewhat likely to change. Therefore I would like to also keep this line. I propose to keep both lines, with `...` between them to signal that other stuff goes in between.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -134,17 +134,20 @@ onwards and Eclipse 2020-3 and onwards.
...
+    [2021-01-24T14:20:16,894][INFO ][o.e.x.s.s.SecurityStatusChangeListener] [runTask-0] Active license is now [BASIC]; Security is enabled
+    <============-> 99% EXECUTING [4m 34s]
+    > :run
+    > IDLE
+    > IDLE
+    > IDLE
 
-But to be honest its typically easier to wait until the console stops scrolling
-and then run `curl` in another window like this:
+Open another terminal window to check if ES is ready to receive requests by running this `curl` command in :
```
Comment (henningandersen) [inline CONTRIBUTING.md:146]: Also limit this to 80 characters.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -134,17 +134,20 @@ onwards and Eclipse 2020-3 and onwards.
...
 
     curl -u elastic:password localhost:9200
 
+> Note: If you need assistance or question, go to our [discussion forum](https://discuss.elastic.co/).
```
Comment (henningandersen) [inline CONTRIBUTING.md:150]:
```suggestion
> Note: If you need assistance or have questions, visit our
[discussion forum](https://discuss.elastic.co/).
```

**Review by henningandersen — CHANGES_REQUESTED** [review]
@ayusopatricia thanks for improving this. I added a few comments, in particular we like to keep this file to 80 characters line width and at least additions should adhere to that.

**Comment by mark-vieira** [comment]
@elasticmachine update branch

**Comment by pugnascotia** [comment]
This PR has had no updates for over a year, so I hope you don't mind if I close it.

**Pull request closed without merging**